tests/TestUtils/BaseTestCase.py

```diff
@@ -66,6 +66,19 @@ def setup_for_chrome(self, mode):
         chrome_options.add_argument(mode)
         chrome_options.add_argument("--no-sandbox")
         chrome_options.add_argument("window-size=1200x600")
+        # Set up download directory
+        # # https://stackoverflow.com/questions/18437816/how-to-find-chrome-download-path-in-java
+        # # http://makeseleniumeasy.com/2018/08/25/how-to-change-default-download-directory-for-chrome-browser-in-selenium-webdriver/
+        prefs = {
+            "download": {
+                "default_directory": os.getcwd()
```

Comment:
@muryanto1 @downiec`test_export_plot_html_via_button` fails because it can't find the generated HTML file, even though it appears to be in the right place after running the test. In fact, the test passes when run a second time (without removing the HTML file). I tried adding a `wait`, thinking that the HTML file was just taking a while to be generated, but that didn't fix it.